src/App.js

```diff
@@ -1,30 +1,15 @@
 /* eslint-disable */
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 /* eslint-enable */
 
 function App() {
```

Comment:
- [ ] It looks like `App.js` is still a function in place of a class-based component, please change this. Good job implementing this change on calculator.js now do it on App.js too. 🐶

Reply:
Hello. please this is the second time you are blocking me for something that is not even in the requirement. Please if it is a must to use class based components please put it in the requirements